specification/05-security_model.adoc

```diff
@@ -414,31 +440,10 @@ from the following adversaries:
     inconsistent with the actual device operation.
 
 5+^| **Mitigations**
-5+a| TBD
+5+a| The `CoVE-IO-T010` threat can be addressed as follows:
 
-
-|===
-
-==== CoVE-IO-T011 - TDI Denial of Service
-
-.CoVE-IO-T011
-[options="header"]
-|===
-| Asset | Threat | Adversary | Scope | Result
-
-| TVM confidential data
-| Denial of service
-| Privileged host software
-| **Not** in scope
-| TVM can not access a TDI that is assigned to it
-
-5+^| **Description**
-5+| A privileged host software component resets or powers down an assigned TDI
-    or its physical device, while the TDI is assigned to a TVM. +
-    The TVM is no longer able to directly access its assigned TDI.
-
-5+^| **Mitigations**
-5+a| TBD
+    - The TVM must accept the DMA translation table explictely.
+    - The TSM must not enable DMA translation table until the TVM accepts it.
```

Comment:
I don't think the TVM should or can accept the DMA translation tables.

```suggestion
    - The RDSM must guarantee that a DMA transaction from one TDI is translated by an IOMMU instance controlled by a TSM that manages the TVM to which the TDI is bound to.
    - The TSM must guarantee that the DMA translation table for a TDI under its control is consistent with the G-stage tables for the TVM the TDI is bound to.
    - The TSM must not enable DMA translation table until the TVM accepts the TDI.
```

Reply:
> I don't think the TVM should or can accept the DMA translation tables.

For GPA->HPA, right, TVM cannot do anything.
But if DMA translation table also has GVA->GPA, then TVM needs to provide input.